__tests__/app.test.js

```diff
@@ -228,6 +228,96 @@ describe("app", () => {
             expect(body.msg).toBe("Not Found");
           });
       });
+      test("POST 400: should return an error if missing either properties on the post request or wrong property names", () => {
+        return request(app)
+          .post("/api/articles/invalid/comments")
+          .send({
+            username: "butter_bridge",
+            bodyyy: "excellent work",
+          })
+          .expect(400)
+          .then(({ body }) => {
+            expect(body.msg).toBe("Bad Request");
+          });
+      });
+    });
+  });
+  describe("PATCH test", () => {
+    describe("/api/articles/:articles_id", () => {
+      test("PATCH 200: should accept a valid patch request, and respond with the updated article, containing the updated votes", () => {
+        return request(app)
+          .patch("/api/articles/7")
+          .send({
+            inc_votes: 7,
+          })
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.updatedArticle).toEqual({
```

Comment:
instead of a brittle matcher like toEqual (here if the data changes shape for whatever reason in each article this test would fail) consider using a more flexible matcher like toMatchObject or similar (where it looks for the properties you tell it to but if there are additional properties it will ignore them)